internal/backends/qt/lib.rs

```diff
@@ -103,6 +103,9 @@ pub const HAS_NATIVE_STYLE: bool = cfg!(not(no_qt));
 /// False if the backend was compiled without Qt so it wouldn't do anything
 pub const IS_AVAILABLE: bool = cfg!(not(no_qt));
 
+pub use qt_widgets::native_style_metrics_init;
+pub use qt_widgets::native_style_metrics_deinit;
```

Comment:
Maybe something like this:

```suggestion
#[cfg(not(no_qt))]
pub use qt_widgets::{native_style_metrics_init, native_style_metrics_deinit}
#[cfg(no_qt)]
pub fn native_style_metrics_init(_ : Pin<&default_backend::native_widgets::NativeStyleMetrics>) { panic!("Qt backend not present"); }
#[cfg(no_qt)]
pub fn native_style_metrics_deinit(_ : Pin<&default_backend::native_widgets::NativeStyleMetrics>) { panic!("Qt backend not present"); }
```